### PR TITLE
adding example folder and foreach example

### PR DESCRIPTION
**Pull request by yaronp68**

### CODE REVIEW

**Comment by nermiller** [comment]
@yaronp68 please sign the CLA when you get the chance. Thank you so much!

**Comment by yaronp68** [comment]
I have signed the CLA but being Redis employee not sure it is required

File: content/rdi/data-transformation/examples/foreach-example.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Write Behind foreach example
+
+The `foreach` section is used in order to explode a list of objects or arrays to rows is a selected target.
+The `foreach` expression is structured in the following way `foreach <field_name>:<JMESPath expression>`.
+The example below uses the `add_field` transformation to prepare the input JSON to the desired structure and then applies `foreach` to write each `order` object as a relational database record using `keys` and `mapping`.
```
Comment (nermiller) [inline content/rdi/data-transformation/examples/foreach-example.md:5]:
```suggestion
The following example uses the `add_field` transformation to prepare the input JSON to the desired structure. Then, it applies `foreach` to write each `order` object as a relational database record using `keys` and `mapping`.
```

Comment (yaronp68) [inline content/rdi/data-transformation/examples/foreach-example.md:5]: The example is null

File: content/rdi/data-transformation/examples/foreach-example.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Write Behind foreach example
+
+The `foreach` section is used in order to explode a list of objects or arrays to rows is a selected target.
```
Comment (elena-kolevska) [inline content/rdi/data-transformation/examples/foreach-example.md:3]:
```suggestion
The `foreach` section is used in order to explode a list of objects or arrays to rows in a selected target.
```

[the remark this answers is not in the document]
Comment (yaronp68) [reply content/rdi/data-transformation/examples/foreach-example.md:3]: yes in typo

**Review by elena-kolevska — APPROVED** [review]

**Pull request merged**